.github/workflows/ci.yml

```diff
@@ -1,23 +1,39 @@
 name: CI
+
 on:
-  - push
-  - pull_request
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - 'README.md'
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'README.md'
```

Comment:
This makes CI only run on certain branch(es), and don't run if **only** README.md is updated